utils/errorreport.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/xsc/services"
+)
+
+// Sends an error report to Coralogix when the Xsc service is enabled.
+// Errors returned by this function typically do not disrupt the flow, as reporting to Coralogix is optional.
+func ReportToCoralogix(serverDetails *config.ServerDetails, errorToReport error, source string) error {
+	log.Info("Sending an error report to Coralogix if Xsc service is enabled...")
```

Comment:
```suggestion
func ReportErrorToXSC(serverDetails *config.ServerDetails, errorToReport error, source string) error {
	log.Debug("Sending an error report to JFrog analytics...")
```
* I will not mention `Coralogix`, we are using the XSC API, internal implementation is not relevant here